web/Views/Landlord/_RegisterPagePartial.cshtml

```diff
@@ -16,63 +16,71 @@
 
     <div class="my-3">
         @Html.HiddenFor(m => m.Unassigned, new { value = Model.Unassigned })
-        <label asp-for="Title" class="form-label"></label>
-        <select asp-for="Title" class="form-select" required>
+        <label asp-for="Title" class="form-label">Title*</label>
+        <select asp-for="Title" class="form-select" required
+                id="Title" onchange="displayIfSelectedValueIsTarget('Other', 'Title', 'titleOther')">
             <option selected disabled value="">Title</option>
             <option value="Mr">Mr</option>
             <option value="Mrs">Mrs</option>
             <option value="Miss">Miss</option>
             <option value="Ms">Ms</option>
             <option value="Dr">Dr</option>
+            <option value="Prof">Prof</option>
+            <option value="Other">Other</option>
         </select>
+        <div id="titleOther" class="my-3" style="@(Model.MembershipId == null ? "display: none;" : "")">

```

Comment:
This should probably be `Model.TitleInput` instead of `Model.MembershipId`? Because at the moment, when you edit a landlord, the "other" input is always shown, even when the title is "Mr" etc.